installer/src/test/java/com/teamscale/profiler/installer/AllPlatformsInstallerTest.java

```diff
@@ -0,0 +1,145 @@
+package com.teamscale.profiler.installer;
+
+import com.teamscale.profiler.installer.utils.MockRegistry;
+import com.teamscale.profiler.installer.utils.MockTeamscale;
+import com.teamscale.profiler.installer.utils.TestUtils;
+import okhttp3.HttpUrl;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Properties;
+
+import static com.teamscale.profiler.installer.utils.UninstallErrorReporterAssert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class AllPlatformsInstallerTest {
+
+	private static final int TEAMSCALE_PORT = 8059;
```

Comment:
Should we use the same mechanism that you have built for the system tests to generate a unique port?